openfe/setup/methods/openmm/equil_rbfe_methods.py

```diff
@@ -0,0 +1,823 @@
+            # These are non-optional settings for this method
+            fix_constraints=True,
+            remove_element_changes=True,
+        )
+
+        # Finally get the positions
+        stateB_positions = _rbfe_utils.topologyhelpers.set_and_check_new_positions(
+            ligand_mappings, stateA_topology, stateB_topology,
+            old_positions=stateA_positions,
+            insert_positions=stateB_openff_ligand.conformers[0],
+            shift_insert=center_offset,
+        )
+
+        ## Create the hybrid topology
+        # Get alchemical settings
+        alchem_settings = self._settings.alchemical_settings
+
+        # Create the hybrid topology factory
+        hybrid_factory = _rbfe_utils.relative.HybridTopologyFactory(
+            stateA_system, stateA_positions, stateA_topology,
+            stateB_system, stateB_positions, stateB_topology,
+            old_to_new_atom_map=ligand_mappings['old_to_new_atom_map'],
+            old_to_new_core_atom_map=ligand_mappings['old_to_new_core_atom_map'],
+            use_dispersion_correction=alchem_settings.use_dispersion_correction,
+            softcore_alpha=alchem_settings.softcore_alpha,
+            softcore_LJ_v2=alchem_settings.softcore_LJ_v2,
+            softcore_LJ_v2_alpha=alchem_settings.softcore_alpha,
+            softcore_electrostatics=alchem_settings.softcore_electrostatics,
+            softcore_electrostatics_alpha=alchem_settings.softcore_electrostatics_alpha,
+            softcore_sigma_Q=alchem_settings.softcore_sigma_Q,
+            interpolate_old_and_new_14s=alchem_settings.interpolate_old_and_new_14s,
+            flatten_torsions=alchem_settings.flatten_torsions,
+        )
+
+        # Add a barostat to the hybrid system
+        hybrid_factory.hybrid_system.addForce(
+            openmm.MonteCarloBarostat(
+                self._settings.barostat_settings.pressure.to(unit.bar).m,
+                self._settings.integrator_settings.temperature.m,
+                self._settings.barostat_settings.frequency.m,
+            )
+        )
+
+        ## Create lambda schedule
+        # TODO - this should be exposed to users, maybe we should offer the
+        # ability to print the schedule directly in settings?
+        lambdas = _rbfe_utils.lambdaprotocol.LambdaProtocol(
+            functions=alchem_settings.lambda_functions,
+            windows=alchem_settings.lambda_windows
+        )
+
+        ## Create the multistate reporter
+        # Get the sub selection of the system to print coords for
+        selection_indices = hybrid_factory.hybrid_topology.select(
+                self._settings.simulation_settings.output_indices
+        )
+
+        # Create the multistate reporter
+        reporter = multistate.MultiStateReporter(
+            self._settings.simulation_settings.output_filename,
+            analysis_particle_indices=selection_indices,
+            checkpoint_interval=self._settings.simulation_settings.checkpoint_interval.m,
```

Comment:
typo + additional option
```suggestion
            checkpoint_interval=self._settings.simulation_settings.checkpoint_interval,
            checkpoint_storage=self._settings.simulation_settings.checkpoint_storage,
```